uiowa/uiowa
# [CCOM] Metatag token

tracks https://github.com/uiowa/uiowa/issues/7522

## To test

1. Sync down something like `neurosurgery.medicine.uiowa.edu`
2. Check that the title of the page (The one in the`<head>` element mirrors the one displayed in your browser tab) now displays both the parent org and the university of iowa.
3. Toggle parent org on and off, check that all combinations of parent org work
4. [Toggle 'Branding options' appearance setting](https://medicineneurosurgery.uiowa.ddev.site/admin/appearance/settings/uids_base) to different values and ensure that it works for all of them with step 3.
5. Try to break this, then comment what you found.
